poker/src/components/CardIssue/CardIssue.module.scss

```diff
@@ -19,23 +19,34 @@
 }
 
 .cardIssue_item h3 {
-  font-size: 1.8rem;
+  font-size: 1.2rem;
+  line-height: 4rem;
+  margin: 0;
   font-weight: 600;
 }
 
 .cardIssue_item p {
   position: absolute;
-  bottom: 0;
+  top: -9px;
```

Comment:
is it necessary to use negative values here&

Reply:
this text can be moved with 'transform: translateY(-9px)', but this is an extra line of code